src/bp-notifications/bp-notifications-settings.php

```diff
@@ -0,0 +1,400 @@
+<?php
+/**
+ * Notifications Settings
+ *
+ * @package BuddyBoss\Notifications
+ * @since BuddyBoss [BBVERSION]
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Get the Notification settings sections.
+ *
+ * @return array
+ * @since BuddyBoss [BBVERSION]
```

Comment:
move since tag after description. Also check at other places.
/**
 * Get the Notification settings sections.
 * 
 * @since BuddyBoss [BBVERSION]
 * 
 * @return array